examples/renderers.cpp

```diff
@@ -1,13 +1,16 @@
 #include <string>
 #include <iostream>
+#include <chrono>
 
 #include "measuro.hpp"
 
 int main(int argc, char * argv[])
 {
     measuro::Registry reg;
 
-    auto count_metric = reg.create_metric(measuro::INT::KIND, "example_count",
+    auto count_metric_1 = reg.create_metric(measuro::INT::KIND, "example_count",
+            "item(s)", "An example count metric", 0);
+    auto count_metric_2 = reg.create_metric(measuro::INT::KIND, "example_count2",
```

Comment:
Exemplifying a `RATE` metric could be helpful -- but you might have that somewhere else.

Reply:
Yeah see `examples/rate.cpp`